app/services/cohort_service.rb

```diff
@@ -48,11 +47,7 @@ def default_range
   end
 
   def cache_key(period)
-    if @with_exclusions
-      "#{self.class}/#{region.cache_key}/#{period.cache_key}/with_exclusions"
-    else
-      "#{self.class}/#{region.cache_key}/#{period.cache_key}"
-    end
+    "#{self.class}/#{region.cache_key}/#{period.cache_key}"
```

Comment:
~When we ship, our cache will temporarily hold the "without exclusions" value in this key. We'll need to warm the cache right after shipping?~

I love our cache versioning ❤️